systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java

```diff
@@ -144,7 +143,6 @@ void testMirrorMaker() {
      * Test mirroring messages by MirrorMaker over tls transport using mutual tls auth
      */
     @ParallelNamespaceTest
-    @Tag(ACCEPTANCE)
```

Comment:
In acceptance, I would check that all the components are at least starting. I know that MM1 is deprecated, but I think that it would be worth checking it in some test in the acceptance profile. And if I read it correctly, this test is the only test we have for MM1.

Reply:
there was till Oauth test with mm1 but i removed that one in favour of keeping metric one as it would be already provisioned there.